### ISSUE TITLE
Steps not clickable 

### ISSUE THREAD

**Issue by shivareddy6** [scorelab/Codelabz#815]
**Describe the bug**
During creating a tutorial, the user is not able to switch between steps non linearly.

**To Reproduce**
1. Go to "My Tutorials" from the user icon menu
2. Create a Tutorial
3. Click on add step and fill in details
4. Not able to click on step options non linearly

**Expected behavior**
User should be able to navigate through steps as he wishes

**Comment by kushalShukla-web** [comment]
bro are you working on it ?

**Comment by shivareddy6** [comment]
Yeah I'v already raised a pr for this but forgot to include the issue in it.
